packages/appstore/src/components/wallet-header/wallet-header-buttons.tsx

```diff
@@ -25,7 +37,15 @@ const WalletHeaderButtons = ({ is_disabled, is_open, btns }: TWalletHeaderButton
                     className={classNames('wallet-header__description-buttons-item', {
                         'wallet-header__description-buttons-item-disabled': is_disabled,
                     })}
-                    onClick={btn.action}
+                    onClick={async () => {
```

Comment:
@thisyahlen-deriv you move this functionality outside the  return for more readability.
```suggestion
                    onClick={btnClickHandler(btn)}
```